src/amo/components/LandingPage/index.js

```diff
@@ -37,61 +37,88 @@ const ICON_MAP = {
 
 export class LandingPageBase extends React.Component {
   static propTypes = {
-    apiAddonType: PropTypes.func.isRequired,
-    contentForType: PropTypes.func,
+    addonTypeOfResults: PropTypes.string,
```

Comment:
I wanted to link this specifically to results because there is already `params.visibleAddonType` which is the `addonType` of the URL (i.e. what was requested). It's a bit confusing so I think it helps to tell them apart.